aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -204,16 +405,69 @@ impl VMChangeSet {
             }
         }
 
-        let write_set = write_set_mut.freeze().map_err(|_| {
-            VMStatus::error(
-                StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR,
-                err_msg("Error when freezing squashed write sets."),
-            )
-        })?;
+        Ok(())
+    }
+
+    /// Squashes `next` writes on top of the write set.
```

Comment:
naming:
`next` -> additional
`squash_writes` -> `squash_additional_writes`. 

And the comment becomes unnecessary.